deps/src/main/java/com/microsoft/azure/sdk/iot/deps/twin/RegisterManager.java

```diff
@@ -123,6 +125,17 @@
     @SerializedName(CAPABILITIES_TAG)
     protected DeviceCapabilities capabilities = null;
 
+    /**
+     * The Digital Twin model id of the device and module
+     * The value will be null for a non-pnp device.
+     * The value will be null for a pnp device until the device connects and registers with the model id.
+     */
+    private static final String MODEL_ID = "digital-twin-model-id";
+    @Expose(serialize = true, deserialize = true)
+    @SerializedName(MODEL_ID)
+    @Setter
+    @Getter
+    protected String modelId = null;
```

Comment:
Is this needed now that modelid sits on device twin?

Reply:
+1 on this comment. This new field should only exist on twin, not identity

Reply:
Synced offline on this. This class is used for Twin and this field exist on twin only.